static/app/views/issueDetails/utils.tsx

```diff
@@ -48,6 +48,30 @@ export function useDefaultIssueEvent() {
   const options = user ? user.options : null;
   return options?.defaultIssueEvent ?? 'recommended';
 }
+/**
+ * Combines two TagValue arrays and combines TagValue.count upon conflict
+ */
+export function mergeTagValues(
+  tagValues1: TagValue[],
+  tagValues2: TagValue[]
+): TagValue[] {
+  const tagValueCollection = tagValues1.reduce<Record<string, TagValue>>(
+    (acc, tagValue) => {
+      acc[tagValue.name] = tagValue;
+      return acc;
+    },
+    {}
+  );
+  tagValues2.forEach(tagValue => {
+    if (tagValueCollection[tagValue.name]) {
+      tagValueCollection[tagValue.name].count += tagValue.count;
+    } else {
+      tagValueCollection[tagValue.name] = tagValue;
+    }
+  });
+
+  return Object.values(tagValueCollection);
```

Comment:
We should also sort the tags here. They should be sorted according to `lastSeen` or `count`, whichever we are using in the original requests. (I think I'd prefer count, but I the default is probably last seen)

Reply:
maybe I'm missing something, but it looks like neither request specifies a sort variable even though the endpoint does accept a sort parameter. I've added a sort param nonetheless and set the default to lastSeen since this is what the backend sorts by default when there is nothing passed in.